src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -6242,4 +6371,186 @@ export class StudyViewPageStore {
             return isMixedReferenceGenome(this.queriedPhysicalStudies.result);
         }
     }
+
+    @observable
+    private _pTreatmentsFilter: AndedPatientTreatmentFilters = { filters: [] };
+
+    @observable
+    private _sTreatmentsFilters: AndedSampleTreatmentFilters = { filters: [] };
```

Comment:
how could you made these private?

Reply:
The public interface is the getters and setters that access these members. Why do you think they should be public?